### PR TITLE
chore(docs): update some docs, also make sure to use jsonify for corr…

### PR DESCRIPTION
**Pull request by Avantol13**
…ect content headers

### Improvements
- use jsonify for response for correct content headers
- update docs to include new google endpoint
- update docs to clarify header for shib

### CODE REVIEW

File: fence/blueprints/data/blueprint.py [hunk 1]
```diff
@@ -55,7 +55,8 @@ def delete_data_file(file_id):
                 return flask.jsonify({"message": message}), status_code
 
             try:
-                return record.delete()
+                delete_info = record.delete()
+                return flask.jsonify(delete_info)
```
Comment (williamhaley) [inline fence/blueprints/data/blueprint.py:59]: I can't tell off-hand, but what status do we return on deletes? Just checking we're not returning a 204 with data now.

Comment (williamhaley) [reply fence/blueprints/data/blueprint.py:59]: And do we also need this on line 98?

Comment (Avantol13) [reply fence/blueprints/data/blueprint.py:59]: yeah this ended up breaking some unit tests, I'm taking a closer look

Comment (Avantol13) [reply fence/blueprints/data/blueprint.py:59]: oh I goofed. The delete function when successful is not returning JSON, it's a tuple with empty text and a 204. and when json is returned, it's jsonified in the delete call. Whoops, so I'm removing this change, it's not necessary

**Review by williamhaley — APPROVED** [review]
:shipit:

**Pull request merged**